avatar/urls.py

```diff
@@ -1,10 +1,11 @@
-from django.conf.urls import url
+from django.conf.urls import include, url
 
 from . import views
 
 urlpatterns = [  # 'avatar.views',
-    url('^add/$', views.add, name='avatar_add'),
-    url('^change/$', views.change, name='avatar_change'),
-    url('^delete/$', views.delete, name='avatar_delete'),
-    url('^render_primary/(?P<user>[\+\w]+)/(?P<size>[\d]+)/$', views.render_primary, name='avatar_render_primary'),
+    url(r'add/', views.add, name='avatar_add'),
```

Comment:
for all the URLs, I'd make sure they're consistent with the previous version, ie start with a `^` and end with a `$`

Reply:
changed except for line 10 where adding a $ to the end raise a warning in python shell.